frontend/lib/src/components/widgets/DataFrame/DataFrame.tsx

```diff
@@ -426,6 +508,8 @@ function DataFrame({
   React.useEffect(() => {
     // Clear cell selections if fullscreen mode changes
     clearCellSelection()
+    // Only run this on changes to the fullscreen mode:
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isFullScreen])
```

Comment:
This is from a long time ago, but there were some weird side effects if the fullscreen mode was changed and cells were selected. Since it doesn't really add value to keep the cell selection, I just cleaned it to avoid dealing with these side effects.

Reply:
alrighty. maybe it's a good idea to add that^ as a brief comment:
`// In the past we saw some weird side-effects, so we decided to clean it when entering fullscreen-mode. If we decide to change this, we have to play around and get to the bottom of it`

Reply:
added 👍